src/rtc/h5_prep.py

```diff
@@ -232,19 +253,86 @@ def populate_metadata_group(h5py_obj: h5py.File,
             [burst_in.sensing_stop.strftime('%Y-%m-%dT%H:%M:%S.%f'),
             'Azimuth stop time of product'],
         'identification/listOfFrequencies':
-             [['A'], 'List of frequency layers available in the product'],  # TBC
+            [['A'],
+             'List of frequency layers available in the product'],  # TBC
         'identification/isGeocoded':
-            [True, 'Flag to indicate radar geometry or geocoded product'],
+            [True,
+             'Flag to indicate radar geometry or geocoded product'],
         'identification/isUrgentObservation':
-            [False, 'List of booleans indicating if datatakes are nominal or urgent'],
+            [False,
+             'List of booleans indicating if datatakes are nominal or urgent'],
         'identification/diagnosticModeFlag':
-            [False, 'Indicates if the radar mode is a diagnostic mode or not: True or False'],
+            [False,
+             'Indicates if the radar mode is a diagnostic mode or not: True or False'],
         'identification/processingType':
-            [processing_type, 'NOMINAL (or) URGENT (or) CUSTOM (or) UNDEFINED'],
+            [processing_type,
+             'NOMINAL (or) URGENT (or) CUSTOM (or) UNDEFINED'],
         'identification/productVersion':
-            [product_version, 'Product version'],
+            [product_version,
+             'Product version'],
+        'identification/DocumentIdentifier':
+            ["https://ceos.org/ard/files/PFS/NRB/v5.5/CARD4L-PFS_NRB_v5.5.pdf",
+             'Product version'],
+        'RTC/grids/sourceDataNumberOfAcquisitions': # placeholder
+            [0,
+             'Number of source data acquisitions'],
+        'RTC/grids/sourceDataAccess': # placeholder
+            ['',
+             'Where the source data can be retrieved'],
+        'RTC/grids/sourceDataProcessingDate': # placeholder for 1.6.6
+            ['0000-00-00T00:00:00.000000',
+             'Processing date'],
+        'RTC/grids/sourceDataProcessingSoftwareVersion': # placeholder for 1.6.6
+            [str(burst_in.ipf_version),
+             'IPF version of the source data'],
+        'RTC/grids/sourceDataProductLevel': # placeholder for 1.6.6
+            [1,
+             'Product level of the source data'],
+        'RTC/grids/sourceDataAzimuthSpacing': # placeholder for 1.6.7
+            [0,
+             'Azimuth time spacing of the source data in seconds'],
+        'RTC/grids/sourceDataRangeSpacing': # placeholder for 1.6.7
+            [0,
+             'Slant range spacing of the source data in meters'],
```

Comment:
Let's move all these datasets to `RTC/metadata/sourceDataInformation` and please remove the prefix `sourceData`. For example: `RTC/grids/sourceDataProductLevel` becomes `RTC/metadata/sourceDataInformation/productLevel`